data/src/main/java/org/oppia/android/data/backends/gae/model/GaeFeedbackReportingAppContext.kt

```diff
@@ -10,16 +10,25 @@ import com.squareup.moshi.JsonClass
 @JsonClass(generateAdapter = true)
 data class GaeFeedbackReportingAppContext(
 
-  @Json(name = "entry_point") val entryPoint: String?,
-  // Maps a string representing the topic progress to the number of topics with that status.
-  @Json(name = "topic_progress") val topicProgress: Map<String, Int>?,
-  @Json(name = "text_size") val textSize: String?,
-  @Json(name = "text_lang") val textLang: String?,
-  @Json(name = "audio_lang") val audioLang: String?,
-  @Json(name = "download_and_update_only_on_wifi") val downloadAndUpdateOnlyOnWifi: Boolean?,
-  @Json(name = "automatically_update_topics") val automaticallyUpdateTopics: Boolean?,
-  @Json(name = "is_admin") val isAdmin: Boolean?,
-  @Json(name = "event_logs") val eventLogs: String?,
-  @Json(name = "logcat_logs") val logcatLogs: String?
+  /** Corresponds to a location in the app that a user can access feedback reporting. */
+  @Json(name = "entry_point") val entryPoint: String,
+  /** Corresponds to the list of topics that the user has completed. */
+  @Json(name = "topic_progress") val topicProgress: List<String?>,
```

Comment:
Does the proto also need to be fixed for this, or was that done already?

Reply:
Thought it was.. fixed now :)